internal/server/evaluation/data/server.go

```diff
@@ -97,7 +97,26 @@ func toEvaluationRolloutType(r flipt.RolloutType) evaluation.EvaluationRolloutTy
 	return evaluation.EvaluationRolloutType_UNKNOWN_ROLLOUT_TYPE
 }
 
-var supportsEntityIdConstraintMinVersion = semver.MustParse("1.38.0")
+type minVersion struct {
+	semver.Version
+	msg string
+}
+
+func (m minVersion) String() string {
+	return m.msg
+}
+
+var (
+	supportsEntityIdConstraintMinVersion = minVersion{
+		Version: semver.MustParse("1.38.0"),
+		msg:     "skipping `entity_id` constraint type to support older client; upgrade client to allow `entity_id` constraint type",
+	}
+
+	supportsDefaultVariantMinVersion = minVersion{
```

Comment:
Should we really worry about it? It probably will not break old clients.

Reply:
you're probably right, i just prefer to err on the side of caution